src/docs/pages/component-docs/components/app-bar/examples/expanded.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component } from '@angular/core';
+
+export const EXPANDED = `<div class="scroll-container">
+    <blui-app-bar variant="expanded">
+        <div>Content</div>
+    </blui-app-bar>
+    <div class="content-body">{{ filler }}</div>
+</div>
+`;
+
+@Component({
+    selector: 'app-expanded-app-bar-demo',
+    template: EXPANDED,
+    styles: [
+        `
+            .scroll-container {
+                width: 100%;
+                max-width: 450px;
+                max-height: 400px;
+                overflow: auto;
+                position: relative;
+            }
+            .content-body {
+                padding: 16px;
+                background: white;
+            }
+        `,
+    ],
+})
+export class ExpandedComponent {
+    filler = `Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas iaculis vulputate arcu, quis vulputate mauris aliquet in. Praesent sollicitudin erat at augue cursus, a scelerisque odio tristique. Aenean quis dui in nisi condimentum viverra eleifend sed odio. Vestibulum a lacinia leo. Proin congue purus vulputate metus vestibulum eleifend. Suspendisse sit amet orci magna. Suspendisse eu odio ut purus ultrices ornare id eu dolor. Cras porttitor semper feugiat. Sed vel ultricies risus, vel dignissim arcu. Vestibulum quis interdum sem. Morbi bibendum tortor ligula, non pellentesque ante ullamcorper sed. Duis ornare felis eget justo faucibus, id tristique felis dignissim. Nullam sapien metus, feugiat egestas convallis eu, vehicula et turpis. Phasellus interdum, justo sit amet maximus pellentesque, eros orci dictum diam, id bibendum risus neque non risus.
```

Comment:
May want to store this filler text as a constant somewhere. You're using it multiple times already we will likely use this lorem Ipsum text again.